web/developer.actyx.com/docs/tutorials/getting-started/first-query.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Running your first query
+hide_title: false
+hide_table_of_contents: false
+image: /images/defaults/default.svg
+---
+
+import { Tabs } from '/src/components'
+
+:::info
+This tutorial is aimed at beginners that already know how to publish events,
+if you're not sure if that is you, take a look at the [previous tutorial](./first-event).
+:::
+
+Before we start, make sure Actyx is running! If you're unsure how to do so,
+you can review how to start Actyx in our [how-to guide](/docs/how-to/local-development/install-actyx#starting-and-stopping-actyx).
+
+To query Actyx you can either use the CLI or the Node Manager —
+don't worry, we will cover both in this tutorial.
+
+## Querying Actyx using the CLI
+
+If you're familiar with CLI applications, it won't be a surprise that the interface to
+query events, is similar to the interface to publish them.
+Let's take a look into the help text!
+
+```
+USAGE:
+    ax events query [FLAGS] [OPTIONS] <NODE> <query>
+
+FLAGS:
+    -h, --help       Prints help information
+    -j, --json       Format output as JSON
+    -V, --version    Prints version information
+    -v
+
+OPTIONS:
+    -i, --identity <FILE>
+            File from which the identity (private key) for
+            authentication is read
+
+ARGS:
+    <NODE>     the IP address or <host>:<admin port> of the node
+               to perform the operation on
+    <query>    event API query (read from file if the argument
+               starts with @)
+```
+
+As you can see, we still need the node, but instead of a `payload`, we're sending a `query` now.
+
+The query is required to be valid AQL code, which we will cover in depth in a later tutorial.
+For this case, we will provide the required AQL query and [explain what it does](#what-does-the-query-do)!
+
+### Running the query
+
+Now that you're up to speed, the command we will be running is the following:
+
+<Tabs>
+<Windows>
+
+```
+.\ax events query localhost "FROM allEvents"
```

Comment:
It wouldn’t make it significantly more difficult to add `--tag a` when publishing the first event and querying `FROM 'a'` here, and I’d think it yields are more empowering experience.